Dockerfile

```diff
@@ -14,6 +12,12 @@ ADD ./dist/*.css /usr/share/nginx/html/dist/
 ADD ./dist/*.png /usr/share/nginx/html/dist/
 ADD ./docker-run.sh /usr/share/nginx/
 
+# needed for gzip after replacements / SWAGGER_FILE
+RUN chgrp -R 0 /usr/share/nginx/html && \
+    chmod -R g=u /usr/share/nginx/html
```

Comment:
I don't assert that there is anything wrong with this approach. However, I note that it is a different pattern than what is used in the [Dockerfile of the swagger-ui repo](https://github.com/swagger-api/swagger-ui/blob/887f45ff1a3da350e7dfda7420926a145e341b72/Dockerfile#L24). 

It's not my call, but I _imagine_ Swagger would prefer to keep these consistent.

Full disclosure: I'm in the biased position of having just put up a [PR with a conflicting change](https://github.com/swagger-api/swagger-editor/pull/2281). It depends on the nginx.conf ([spot 1](https://github.com/swagger-api/swagger-editor/pull/2281/files#diff-360c358e5fc4de52699ff11376043b7c67ebccc164c8dd9bc2b025782d681279R10-R12), [spot 2](https://github.com/swagger-api/swagger-editor/pull/2281/files#diff-360c358e5fc4de52699ff11376043b7c67ebccc164c8dd9bc2b025782d681279R27-R29)) to enable altering the BASE_URL via environment variable.